Source/CarthageKit/Xcode.swift

```diff
@@ -98,7 +98,9 @@ public func locateProjectsInDirectory(directoryURL: NSURL) -> SignalProducer<Pro
 				.carthage_enumeratorAtURL(directoryURL.URLByResolvingSymlinksInPath!, includingPropertiesForKeys: [ NSURLTypeIdentifierKey ], options: enumerationOptions, catchErrors: true)
 				.map { _, URL in URL }
 				.filter { URL in
-					return !directoriesToSkip.contains { $0.hasSubdirectory(URL) }
+					return !directoriesToSkip.contains {
+						return NSFileManager.defaultManager().carthage_hasSubdirectory($0, otherURL: URL);
```

Comment:
Can you create a local constant for this file manager to reuse it above too and remove the duplication?